enterprise/frontend/src/embedding-sdk/index.ts

```diff
@@ -3,6 +3,8 @@ import "./lib/polyfill/use-sync-external-store";
 
 import "metabase/lib/dayjs";
 
+import "ee-plugins";
```

Comment:
We can load all plugins now. But it won't do anything, it just won't complain about the unknown `token-feature` anymore.

This works because a change in `enterprise/frontend/src/metabase-enterprise/settings/index.ts`

https://github.com/metabase/metabase/blob/86401405a620ef3f86506960d5198e98c9d7066a/enterprise/frontend/src/metabase-enterprise/settings/index.ts#L7-L11